app/controllers/katello/api/v2/content_export_incrementals_controller.rb

```diff
@@ -0,0 +1,99 @@
+module Katello
+  class Api::V2::ContentExportIncrementalsController < Api::V2::ApiController
+    before_action :fail_if_not_pulp3, :only => [:version, :library]
+    before_action :find_exportable_organization, :only => [:library]
+    before_action :find_exportable_content_view_version, :only => [:version]
+    before_action :find_library_export_view, :only => [:library]
+    before_action :find_history, :only => [:version, :library]
+
+    api :POST, "/content_export_incrementals/version", N_("Performs an incremental-export of a content view version.  Relevant only for Pulp 3 repositories")
+    param :id, :number, :desc => N_("Content view version identifier"), :required => true
+    param :destination_server, String, :desc => N_("Destination Server name, required for Pulp3"), :required => false
```

Comment:
Do we need to mention pulp3 for `destination_server` The less we mention pulp in any context (particularly in APIs) the better

Reply:
Typo shouldnt ve been there.